app/assets/stylesheets/_billables.scss

```diff
@@ -26,10 +26,8 @@
 
     max-width: 90em;
 
-    .submit-billable-entries {
-      @include media($laptop-screen) {
-        opacity: 0;
-      }
+    #submit-billable-entries-test {
+      opacity: .0;
```

Comment:
this is a hidden button to circumvent some issues with the Capybara webkit driver that doesn't play nice with our subdomains.

Was already there, just made it easier to find.